### ISSUE TITLE
`meson install` should install gcc runtime .dll's next to nix.exe

### ISSUE THREAD

**Issue by volth** [nix-windows/nix#25]
`meson install` should install gcc runtime .dll's (`libgcc_s_dw2-1.dll`, `libstdc++.dll`, `libwinpthread-1.dll`) next to `nix.exe`.

Even if we build Nix with `-static-libstdc++`, `-static-libgcc`, ..  `libcurl-1.dll` and other dep dll's do need gcc runtime .dll's.

Alternatively, we can build fully static `nix.exe`, which looks like a good option, as there is only one exe (`nix-build.exe`, `nix-instantiate.exe`, `nix-store.exe`, ... went away in upstream, they are just symlinks to `nix` executable)

**Comment by Ericson2314** [comment]
FYI. Passing `-Ddefault_library=static` to Meson will make it build statically.

More broadly, I'm not sure whether copying/symlinking the DLLs there ought to be Meson's or Windows responsibility, but agreed something has to do it.

**Comment by volth** [comment]
> FYI. Passing `-Ddefault_library=static` to Meson will make it build statically.

No, it is only about `libexpr.dll`, `libutil.dll`, and other nix's own libraries.
